### ISSUE TITLE
Crash with VarusBikerEdition mod

### ISSUE THREAD

**Issue by Try** [Try/OpenGothic#211]
This mods cause game to crash at `init_global`, because world is not yet created here.
Link: https://worldofplayers.ru/threads/38143/

```
// Daedalus callstack
INIT_GLOBAL
MEM_INITALL
MEM_INITLABELS
MEM_PTRTOINST
MEM_ASSIGNINST
MEM_WRITEINT
MEMINT_ASSIGN
MEMINT_PREPAREASSIGNMENTS
MEMINT_GETMEMHELPER
WLD_INSERTNPC
```
This call-stack look a lot like Ikarus/LeGo. In anyway - good content to debug and study

**Comment by Try** [comment]
Checked mod executable:
 - [x] Ikarus
 - [ ] LeGo

Use case is seemingly small: `Pedro` always scanning for a door and that's it.
Gothic sources crashed, in the attempt to decompile Gothic.dat

**Comment by Try** [comment]
fixed by 27f10cd
